### DISCUSSION TITLE
Changing the properties of mesh layer

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#249] [General]
Is there a way to change the properties of mesh layer?

**Comment by djlittl** [comment]
I'm not sure I understand the question.  But I think the answer is that in order to change the properties for a set of nodes (e.g., a layer), you need to have those nodes in different block from the other nodes.  Then you can set the properties on that block.

**Reply by oldninja** [reply]
That's exactly what I have done in the past with some FGM tests.
